docs/spec/slashing/tombstone.md

```diff
@@ -0,0 +1,77 @@
+# Staking Tombstone
+
+## Abstract
+
+In the current implementation of the `slashing` module, when the consensus engine informs the state machine of a validator's consensus fault, the validator is partially slashed, and put into a "jail period", a period of time in which they are not allowed to rejoin the validator set.  However, because of the nature of consensus faults and ABCI, there can be a delay between an infraction occuring, and evidence of the infraction reaching the state machine (this is one of the primary reasons for the existence of the unbonding period).
+
+In the current system design, once a validator is put in the jail for a consensus fault, after the `JailPeriod` they are allowed to send a transaction to `unjail` themselves, and thus rejoin the validator set.
+
+One of the "design desires" of `slashing` module is that if multiple infractions occur before evidence is executed (and a validator is put in jail), they should only be punished for single worst infraction, but not cumulatively.  For example, if the sequence of events is:
```

Comment:
We should explicitly lay out a list of design desires for slashing and reference it here.

Reply:
That seems like it should be _somewhere_ in the slashing spec, but not necessarily in this tombstone spec.